src/Compilers/VisualBasic/Portable/Symbols/Metadata/PE/PEParameterSymbol.vb

```diff
@@ -605,6 +610,30 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols.Metadata.PE
             End Get
         End Property
 
+        Friend Overrides ReadOnly Property CallerArgumentExpressionParameterIndex As Integer
+            Get
+                If _lazyCallerArgumentExpressionParameterIndex = UninitializedCallerArgumentExpressionParameterIndex Then
+                    Debug.Assert(Not _handle.IsNil)
+
+                    Dim attribute = PEModule.FindTargetAttribute(_handle, AttributeDescription.CallerArgumentExpressionAttribute)
+                    Dim parameterName As String = Nothing
+                    If attribute.HasValue AndAlso PEModule.TryExtractStringValueFromAttribute(attribute.Handle, parameterName) Then
+                        Dim parameters = ContainingSymbol.GetParameters()
+                        For i = 0 To parameters.Length - 1
+                            If parameters(i).Name.Equals(parameterName, StringComparison.Ordinal) Then
```

Comment:
>parameters(i).Name.Equals(parameterName, StringComparison.Ordinal) [](http://example.com/codeflow?start=31&length=66)

I do not think I agree with this change. Languages are already different in terms of case-sensitivity. I believe, the difference in behavior can be observed with named arguments, for example. And probably with other similar scenarios. @333fred, if you believe language rules should not apply for this scenario, please bring this to LDM

Reply:
@AlekseyTs Should I add a prototype until this is discussed in LDM?

My 2 cents: The *current* language case-insensitivity doesn't affect consuming the code in C#, but in this case, it will affect it as @333fred stated. So I'm leaning towards being case-sensitive.

Reply:
> Should I add a prototype until this is discussed in LDM?

I would be fine with behavior on which I signed off (i.e. case-insensitive behavior consistent with named arguments) and a prototype comment to confirm that at LDM. I personally fine with putting the burden of interop with other languages (if one is required) on the developer and leave it to him/her to match the case.

Reply:
I do disagree, but as long as we have a prototype comment to follow up on this I'm ok with merging.